scripts/restart.sh

```diff
@@ -22,23 +22,22 @@ function schedule_restart() {
                     send_restart_now_notification
                 fi
                 break
-            else 
+            else
                 ew ">>> Server has still players"
             fi
-            time=$(date '+%H:%M:%S')
-            rconcli "broadcast ${time}-AUTOMATIC-RESTART-IN-$counter-MINUTES"
+            rconcli broadcast "$(get_time) AUTOMATIC RESTART IN $counter MINUTES"
```

Comment:
function get_time() is defined in rcon.sh
The restart-script doesnt include a source of this file, so this will never be reached.

Reply:
Sourced `/includes/rcon.sh` at the top of the file.